configure.ac

```diff
@@ -0,0 +1,72 @@
+dnl require autoconf 2.60 (AS_ECHO/AS_ECHO_N)
+AC_PREREQ([2.60])
+define(_CLIENT_VERSION_MAJOR, 0)
+define(_CLIENT_VERSION_MINOR, 93)
+define(_CLIENT_VERSION_BUGFIX, 0)
+define(_CLIENT_VERSION_AUTOINCREMENT, 0)
+define(_CLIENT_VERSION_BUILD, 0)
+define(_CLIENT_VERSION_IS_RELEASE, false)
+define(_COPYRIGHT_YEAR, 2015)
+AC_INIT([Armory],[_CLIENT_VERSION_MAJOR._CLIENT_VERSION_MINOR._CLIENT_VERSION_BUGFIX._CLIENT_VERSION_AUTOINCREMENT],[bugs@bitcoinarmory.com])
```

Comment:
I don't think this email address is accurate. Not sure what to replace it with, though.